### PR TITLE
Adds a looksLikeRtcp utility method

### PR DESCRIPTION
**Pull request by bgrozev**
Also moves RTCPHeaderUtils to util.RTCPUtils

### CODE REVIEW

**Comment by gpolitis** [comment]
test this please

**Comment by bgrozev** [comment]
test this please

File: src/org/jitsi/impl/neomedia/AbstractRTPPacketPredicate.java [hunk 1]
```diff
@@ -53,43 +53,22 @@ public AbstractRTPPacketPredicate(boolean rtcp)
...
-                else
-                {
-                    result = false;
-                }
-            }
-            else
-            {
-                result = false;
-            }
+        if (RTCPUtils.looksLikeRtcp(
```
[the remark this answers is not in the document]
Comment (bgrozev) [reply src/org/jitsi/impl/neomedia/AbstractRTPPacketPredicate.java]: When we are actually parsing the packet. Now that I think about it more, I agree that `isRtcp` would make sense (and I don't like `looksLikeRtcp`) as long as the documentation is clear. Do you think I should rename it?

[the remark this answers is not in the document]
Comment (bbaldino) [reply src/org/jitsi/impl/neomedia/AbstractRTPPacketPredicate.java]: yeah i think `isRtcp` would be good.  i think the documentation as you have it is good.

**Pull request merged**